assets/newsletter-section.css

```
@@ -2,6 +2,49 @@
   padding-right: calc(4rem / var(--font-body-scale));
   padding-left: calc(4rem / var(--font-body-scale));
 }
+.newsletter__wrapper-new {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 5rem;
+}
+.form-container {
+  display: flex;
+  height: 100%;
+  margin: 0 !important;
+}
+.input-container {
+}
+.field__input-new {
+  height: 100%;
+  padding: 1em 2em;
+  border-radius: 5px;
+  outline: none;
+  padding-right: 8rem;
```

```suggestion
  padding-right: 8rem;
  border: none;
  height: 54px
```

The design does not have any border colour included but yours does. We need to ensure it looks the same. And the height is taller in their design.